### DISCUSSION TITLE
How to link to page directory instead of specific file

### DISCUSSION THREAD

**Discussion by mattkreikemeier** [parcel-bundler/parcel#4904] [General]
I'm converting a site to use parcel and am trying to figure out how to link to a directory instead of directly to the index.html in that directory.

For example, my existing nav is like this...
```
<ul>
    <li class="mainNav__item">
      <a href="/">Home</a>
    </li>
    <li class="mainNav__item">
      <a href="/work/">Work</a>
    </li>
    <li class="mainNav__item">
      <a href="/team/">Team</a>
    </li>
    <li class="mainNav__item">
      <a href="/contact/">Contact</a>
    </li>
  </ul>
```
This doesn't seem to work.  When I fire up the local server and click the nav item, the URL updates, but it doesn't actually go to the page.  However, if I change all of the hrefs to "/work/index.html" for example (explicitly pointing to index.html), then it works perfectly. 

I can certainly make that change, but I'd prefer not to and figure there must be something I'm missing to just link to the directory.

Thanks!

**Comment by northern-mindset** [comment]
@callmegoon Have you ended up with any solution?

**Reply by mattkreikemeier** [reply]
I did not 😩  — I haven't looked at it in a while though. I also have not tried Parcel 2 which just came out of beta.

**Reply by adrian5** [reply]
This issue appears to persist with Parcel 2.
